### DISCUSSION TITLE
Welcome to BotUI Discussions!

### DISCUSSION THREAD

**Discussion by moinism** [botui/botui#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sandeepsuri** [comment]
Is legacy botui scripts still available to use?

**Reply by moinism** [reply]
Sorry for the late response. Yes, the `master` branch still contains the legacy code: https://github.com/botui/botui/tree/master/build
